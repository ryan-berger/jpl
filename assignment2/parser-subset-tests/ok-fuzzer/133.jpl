read image "" to A
read image "" to X0M_7P.
read image "rd" to b2x_3G.
read image "" to f
time show 4
time write image 016 to "&>"
print "A"
write image 77 to ""
read image "$" to C
print ""
show V5Z.3r_
show j9u_
write image j() to "j"
