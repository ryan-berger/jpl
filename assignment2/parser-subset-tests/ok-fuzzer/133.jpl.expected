(ReadImageCmd "" (VarArgument A))
(ReadImageCmd "" (VarArgument X0M_7P.))
(ReadImageCmd "rd" (VarArgument b2x_3G.))
(ReadImageCmd "" (VarArgument f))
(TimeCmd (ShowCmd (IntExpr 4)))
(TimeCmd (WriteImageCmd (IntExpr 16) "&>"))
(PrintCmd "A")
(WriteImageCmd (IntExpr 77) "")
(ReadImageCmd "$" (VarArgument C))
(PrintCmd "")
(ShowCmd (VarExpr V5Z.3r_))
(ShowCmd (VarExpr j9u_))
(WriteImageCmd (CallExpr j) "j")
